### DISCUSSION TITLE
Runtime options

### DISCUSSION THREAD

**Discussion by wimjongman** [eclipse-platform/eclipse.platform#246] [Q&A] [answered]
I want to find the official source of the runtime / startup options, but no matter how I search on google, I can't find an official source. What am I doing wrong or where did it go?

https://www.google.com/search?q=eclipse+runtime+options+osgi
https://www.google.com/search?q=eclipse+startup+options+osgi
https://www.google.com/search?q=eclipse+runtime+options+osgi+ini
https://www.google.com/search?q=eclipse+runtime+options+osgi+properties

There is an old IBM page here: https://www.ibm.com/docs/pl/rsar/9.5?topic=information-runtime-options

**Comment by akurtakov** [comment]
Are you looking for https://github.com/eclipse-platform/eclipse.platform.common/blob/master/bundles/org.eclipse.platform.doc.isv/reference/misc/runtime-options.html ?

**Reply by wimjongman** [reply]
Thanks, but no. I want to find a web page with this content.

**Comment by iloveeclipse** [comment, marked as answer]
https://help.eclipse.org/latest/index.jsp?topic=/org.eclipse.platform.doc.isv/reference/misc/runtime-options.html

**Reply by wimjongman** [reply]
Yes, this is what I am looking for. 

Why can't this be found on google?  It looks like this eclipse content is no longer being indexed. @eclipsewebmaster, can you please comment?

**Reply by iloveeclipse** [reply]
See https://www.google.com/search?q=eclipse+help+runtime+options+osgi

Note "No information is available for this page."

![image](https://user-images.githubusercontent.com/964108/198239195-97ca61ba-4008-4918-980b-3a1a68ca8afa.png)

Google says to it:
> It means that the website prevented Google from creating a page description, but didn't actually hide the page from Google.
> If you own this site you can improve this result either by letting Google read the page to create a good description, or by hiding the page entirely from Google Search results. Read below to learn how.

**Reply by wimjongman** [reply]
Yes, pretty weird. On top of that, that Eclipse Galileo link goes nowhere.

**Reply by wimjongman** [reply]
I filed https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/2160
